test/unit/lib/plugins/aws/invoke.test.js

```diff
@@ -7,8 +7,10 @@ const proxyquire = require('proxyquire');
 const AwsProvider = require('../../../../../lib/plugins/aws/provider');
 const Serverless = require('../../../../../lib/Serverless');
 const { getTmpDirPath } = require('../../../../utils/fs');
+const runServerless = require('../../../../utils/run-serverless');
```

Comment:
I've created a base for new tests in this file (was just merged to master, and also opened issue that welcomes the refactor of tests in this file: https://github.com/serverless/serverless/issues/8662)

I propose to add following tests here: https://github.com/serverless/serverless/blob/b72a910604b4e7d3dc3da267f47e5ed41a34d551/test/unit/lib/plugins/aws/invoke.test.js#L456

- ``should support `--context` param``
- ``should support `-contextPath` param``

And simply unskip and configure missing parts.